### PR TITLE
Add networking.k8s.io/v1 support

### LINKED ISSUE #2: Ingress is written for networking.k8s.io/v1beta1
This means warnings are thrown from k8s 1.19+

From k8s 1.22 (when it is released) the chart will stop working altogether.

### PR DESCRIPTION
**Pull request by fsmaia**
Closes #2

### CODE REVIEW

**Review by tico24 — COMMENTED** [review]
Probably worth adjusting the KinD version back to 'latest' (just delete the image tag) then the tests will tell us if this works.
Other than that, it looks good to my untrained eye.

File: .github/workflows/lint-test.yml [hunk 1]
```diff
@@ -43,8 +43,8 @@ jobs:
       - name: Create kind cluster
         uses: helm/kind-action@v1.1.0
         with:
```
Comment (tico24) [inline .github/workflows/lint-test.yml:45]: Delete the whole:
'''
with:
  node_image:....
'''
block

File: charts/sorry-cypress/templates/ingress-dashboard.yml [hunk 2]
```diff
@@ -34,8 +37,18 @@ spec:
...
+            backend:
+              service:
+                name: {{ $fullName }}-api
+                port:
+                  number: {{ $servicePort }}
+          {{- else }}
           - backend:
-              serviceName: {{ $fullName }}-dashboard
-              servicePort: {{ $svcPort }}
+              serviceName: {{ $fullName }}-api
```
Comment (fsmaia) [inline charts/sorry-cypress/templates/ingress-dashboard.yml:50]: A wild bug has appeared!

File: charts/sorry-cypress/templates/ingress-director.yml [hunk 3]
```diff
@@ -34,8 +37,18 @@ spec:
...
+            backend:
+              service:
+                name: {{ $fullName }}-api
+                port:
+                  number: {{ $servicePort }}
+          {{- else }}
           - backend:
-              serviceName: {{ $fullName }}-director
-              servicePort: {{ $svcPort }}
+              serviceName: {{ $fullName }}-api
```
Comment (fsmaia) [inline charts/sorry-cypress/templates/ingress-director.yml:50]: A wild bug has appeared!

**Pull request merged**